doc/rules/import/fully_qualified_strict_types.rst

```diff
@@ -29,6 +29,16 @@ Allowed types: ``bool``
 
 Default value: ``false``
 
+``phpdoc_tags``
+~~~~~~~~~~~~~~~
+
+Collection of PHPDoc annotation tags where FQCNs should be processed. As of now
+only simple tags with ``@tag \F\Q\C\N`` format are supported (no complex types).
+
+Allowed types: ``array``
+
+Default value: ``['param', 'phpstan-param', 'phpstan-property', 'phpstan-property-read', 'phpstan-property-write', 'phpstan-return', 'phpstan-var', 'property', 'property-read', 'property-write', 'psalm-param', 'psalm-property', 'psalm-property-read', 'psalm-property-write', 'psalm-return', 'psalm-var', 'return', 'see', 'throws', 'var']``
```

Comment:
LGTM, I would personally remove `@see` which is required to be FQCN in some coding standards and very often is not a FQCN alone, it is very often a method name, which is not a "type" which this fixer is about.

Reply:
Yes, but as a second condition fixer matches simple FQCNs with a regex, so URLs or methods won't be processed 🙂.